### PR TITLE
Added icons for arrow keys

### PR DESCRIPTION
**Pull request by barklund**
## Summary

As it says on the tin, this adds an icon for arrow keys rather than depending on an emoji (which might get replaced in WP by some other image).

This is a follow-up to #4525.

### CODE REVIEW

File: assets/src/edit-story/components/keyboardShortcutsMenu/constants.js [hunk 1]
```diff
@@ -32,8 +37,8 @@ export const SPECIAL_KEYS = {
   ENTER: { symbol: '⏎', title: __('Enter', 'web-stories') },
   SHIFT: { symbol: '⇧', title: __('Shift', 'web-stories') },
   OPTION: { symbol: '⌥', title: __('Option', 'web-stories') },
-  ALT: 'Alt',
+  ALT: __('Alt', 'web-stories'),
```
Comment (barklund) [inline assets/src/edit-story/components/keyboardShortcutsMenu/constants.js]: @swissspidy, do you want a translator comment or context on this one? And maybe the other keys as well? They all seem fairly ambiguous to me.

Comment (swissspidy) [reply assets/src/edit-story/components/keyboardShortcutsMenu/constants.js]: context everywhere sounds reasonable 👍

File: assets/src/edit-story/components/keyboardShortcutsMenu/constants.js [hunk 2]
```diff
@@ -19,6 +19,11 @@
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import { Up, Down } from './styled';
```
Comment (miina) [inline assets/src/edit-story/components/keyboardShortcutsMenu/constants.js:25]: Are we shifting to keeping `styled` separately?

**Review by miina — APPROVED** [review]
LGTM!

Would be nice to have a snapshot from the keyboard menu for Percy, assuming it doesn't exist since it isn't among the screenshots to review.

**Review by mariano-formidable — APPROVED** [review]
Pulled it down and opened up the menu.  No more emoji arrows 🎉

File: assets/src/edit-story/components/keyboardShortcutsMenu/constants.js (hunk 2 above)
Comment (barklund) [reply assets/src/edit-story/components/keyboardShortcutsMenu/constants.js:25]: It's been used a few times. I don't think we should do this automatically, but if need be, yes.

File: assets/src/edit-story/components/keyboardShortcutsMenu/constants.js (hunk 1 above)
Comment (barklund) [reply assets/src/edit-story/components/keyboardShortcutsMenu/constants.js]: @swissspidy, what context message would be appropriate? I'm not familiar with the best-practice lingo here. `'The keyboard key "Enter"'` or?

Comment (swissspidy) [reply assets/src/edit-story/components/keyboardShortcutsMenu/constants.js]: @barklund that works!

Comment (barklund) [reply assets/src/edit-story/components/keyboardShortcutsMenu/constants.js]: Done in 62e81c2c4.

**Comment by csossi** [comment]
Verified in QA

**Comment by csossi** [comment]
Verified in QA

**Pull request merged**